sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Configs.java

```diff
@@ -600,4 +609,60 @@ public static CosmosMicrometerMetricsConfig getMetricsConfig() {
 
         return CosmosMicrometerMetricsConfig.fromJsonString(metricsConfig);
     }
+
+    public static PartitionLevelCircuitBreakerConfig getPartitionLevelCircuitBreakerConfig() {
+        String partitionLevelCircuitBreakerConfig =
+            System.getProperty(
+                PARTITION_LEVEL_CIRCUIT_BREAKER_CONFIG,
+                firstNonNull(
+                    emptyToNull(System.getenv().get(PARTITION_LEVEL_CIRCUIT_BREAKER_CONFIG)),
+                    DEFAULT_PARTITION_LEVEL_CIRCUIT_BREAKER_CONFIG));
+
+        return PartitionLevelCircuitBreakerConfig.fromJsonString(partitionLevelCircuitBreakerConfig);
+    }
+
+    public static int getStaleCollectionCacheRefreshRetryCount() {
+        String valueFromSystemProperty = System.getProperty(STALE_COLLECTION_CACHE_REFRESH_RETRY_COUNT);
+
+        if (valueFromSystemProperty != null && !valueFromSystemProperty.isEmpty()) {
+            return Integer.valueOf(valueFromSystemProperty);
+        }
+
+        String valueFromEnvVariable = System.getenv(STALE_COLLECTION_CACHE_REFRESH_RETRY_COUNT);
+        if (valueFromEnvVariable != null && !valueFromEnvVariable.isEmpty()) {
+            return Integer.valueOf(valueFromEnvVariable);
+        }
+
+        return DEFAULT_STALE_COLLECTION_CACHE_REFRESH_RETRY_COUNT;
+    }
+
+    public static int getStaleCollectionCacheRefreshRetryIntervalInSeconds() {
+        String valueFromSystemProperty = System.getProperty(STALE_COLLECTION_CACHE_REFRESH_RETRY_INTERVAL_IN_SECONDS);
+
+        if (valueFromSystemProperty != null && !valueFromSystemProperty.isEmpty()) {
```

Comment:
StringUtils.isNotEmpty("valueFromSystemProperty")?

Reply:
Fixed in next iteration.